modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -53,14 +55,15 @@ class ApiClient(object):
         'datetime': datetime.datetime,
         'object': object,
     }
+    _pool = None
 
     def __init__(self, configuration=None, header_name=None, header_value=None,
-                 cookie=None):
+                 cookie=None, pool_threads=1):
```

Comment:
It looks like a breaking change. I suggest leaving `=None` and still using default number of threads. It shouldn't be a problem because the pool will be created on the first request.

Reply:
Thanks for the feedback.  I'll set `pool_thread=None`

cc @minrk

Reply:
FWIW, this is not a breaking change, though it is a change in the default *performance* of concurrent async requests from the same client object. Tying concurrent IO to CPU count doesn't make a lot of sense to me, especially in Python. I'd argue that a fixed default pool size (if not 1, maybe 4 or 10) would be more logical than spawning 64 threads on a big machine, and would result in more consistent behavior.

Reply:
@minrk I agree with you. This default may not work well on a huge server but it's default, it may be optimal in general, this calculation may be changed in the future releases of Python. Someone can base on this behavior and it'd be a breaking change for him/her.

Reply:
What about using `pool_thread=None` for current master to make it backward-compatible and using `pool_threads=1` in 4.0.x (a major release with breaking changes scheduled to be released next month)?

Reply:
@wing328 Sounds good to me.